cirq/protocols/circuit_diagram_info.py

```diff
@@ -120,6 +121,19 @@ def __repr__(self):
                 self.precision,
                 self.qubit_map))
 
+    def as_dict(self):
```

Comment:
Replace this with a `__copy__` method or `copy` method and mutation of the copy.

Reply:
Done.